test/unit/doctor_test.exs

```diff
@@ -1,22 +0,0 @@
-defmodule PRM.Unit.DoctorTest do
```

Comment:
I am removing this test because this functionality is now tested by a test suite for `PRM.DoctorAPI`